src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/simulations/CCD_PerformanceRegression.scala

```diff
@@ -287,24 +288,35 @@ class CCD_PerformanceRegression extends Simulation  {
   }
 
 	setUp(
-		// API_ProbateCreateCase.inject(simulationProfile(testType, probateTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_CMCCreateCase.inject(simulationProfile(testType, cmcTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_DivorceCreateCase.inject(simulationProfile(testType, divorceTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_IACCreateCase.inject(simulationProfile(testType, iacTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-    // CaseActivityListScn.inject(rampUsers(500) during (10 minutes)),		
-		// CaseActivityScn.inject(rampUsers(500) during (10 minutes)),
-    // CCDSearchView.inject(rampUsers(200) during (20 minutes)),		
-		// CCDElasticSearch.inject(rampUsers(300) during (20 minutes)), //300 during 20
-
-    API_CMCCaseEvents.inject(simulationProfile(testType, cmcTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-
-
-		// CaseActivityListScn.it(simulationProfile(testType, searchTargetPerHour/searchRepeatsPerUser, numberOfPipelineUsers)).pauses(pauseOption),  	
+     //simulation for cdm-test-performance repo
+		 API_ProbateCreateCase.inject(simulationProfile(testType, probateTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
+		 API_CMCCreateCase.inject(simulationProfile(testType, cmcTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
+		 API_DivorceCreateCase.inject(simulationProfile(testType, divorceTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
+		 API_IACCreateCase.inject(simulationProfile(testType, iacTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
+     CaseActivityListScn.inject(rampUsers(500) during (10 minutes)),
+		 CaseActivityScn.inject(rampUsers(500) during (10 minutes)),
+     CCDSearchView.inject(rampUsers(200) during (20 minutes)),
+		 CCDElasticSearch.inject(rampUsers(300) during (20 minutes)), //300 during 20
+
+    //smoke test to be pushed to the ccd_GC_profiling branch only.  Avoids having to amend the full test settings above
```

Comment:
This smoke test won't work as expected unless run in 'debug' mode. If a user tries to kick off this simulation, it will run 4 of the scenarios at full load.

Reply:
@jonsoloway-hmcts good spot - i was running it like that in the 'smoke test' on Friday.  Addressed that in follow up commit.